src/ln/channelmonitor.rs

```diff
@@ -763,6 +769,23 @@ impl ChannelMonitor {
 		}
 	}
 
+	pub(super) fn provide_rescue_remote_commitment_tx_info(&mut self, commitment_number: u64, their_revocation_point: PublicKey) {
+		self.their_cur_revocation_points = Some((commitment_number, their_revocation_point, None));
```

Comment:
Hmm, this seems redundant now (and easier to remove this than worry about whether its right in every case, no :p).

Reply:
Derive private key at this point too, diff should be even smaller in check_spend_remote :)